docs/install.md

```diff
@@ -323,3 +323,13 @@ model = GroqModel('llama-3.1-70b-versatile', api_key='your-api-key')
 agent = Agent(model)
 ...
 ```
+
+### Ollama
+
+To use [Ollama](https://ollama.com/), you must first download the Ollama client, and then download a model.
+
+You must also ensure the Ollama server is running when trying to make requests to it
```

Comment:
I think it might be worth including an example, e.g.:

``````md
With `ollama` installed, you can run the server with the model you want to use:

```bash title="terminal-run-ollama"
ollama run llama3.2
```
(this will pull the `llama3.2` model if you don't already have it downloaded)

Then run your code, here's a minimal example:

```py title="ollama_example.py"
from pydantic_ai import Agent
from pydantic import BaseModel

class CityLocation(BaseModel):
    city: str
    country: str

agent = Agent('ollama:llama3.2', result_type=CityLocation)

result = agent.run_sync('Where the olympics held in 2012?')
print(result.data)
print(result.cost())
```

See [`OllamaModel`][pydantic_ai.models.ollama.OllamaModel] for more information.
``````

Reply:
This makes sense, but does this make more sense in the install guide, or in the `docs.api.models.ollama` file (similar to Vertex ai example)

Reply:
ye, maybe the example should go in the `ollama.md` file.